protocol.html

```diff
@@ -422,7 +422,7 @@ <h3 property="schema:name skos:prefLabel">Terminology</h3>
 
                   <dl>
                     <dt about="#notification-channel" property="skos:prefLabel" typeof="skos:Concept"><dfn id="notification-channel">notification channel</dfn></dt>
-                    <dd about="#notification-channel" property="skos:definition">A <em>notification channel</em> is an abstract thing which is identified by a URI and whose properties describe the available communication interface and features.</dd>
+                    <dd about="#notification-channel" property="skos:definition">A <em>notification channel</em> is an abstract thing whose properties describe the available communication interface and features.</dd>
```

Comment:
The reason for this change is to allow bnodes. An alternative could be:

```suggestion
                    <dd about="#notification-channel" property="skos:definition">A <em>notification channel</em> is an abstract thing which is identified by a URI or a blank node whose properties describe the available communication interface and features.</dd>
```